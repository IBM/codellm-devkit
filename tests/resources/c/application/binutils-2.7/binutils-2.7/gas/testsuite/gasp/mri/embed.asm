embed	macro	label
addr&&label	dc.l	label
	endm

	embed	foo
