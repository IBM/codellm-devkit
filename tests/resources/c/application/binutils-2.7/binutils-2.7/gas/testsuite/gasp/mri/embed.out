;embed	macro	label
;addr&&label	dc.l	label
;	endm
;

;	embed	foo
;addrfoo	dc.l	foo
addrfoo:	dc.l	foo
;